library/traefik

```diff
@@ -1,7 +1,8 @@
 # maintainer: Emile Vauge <emile@vauge.com> (@emilevauge)
 # maintainer: Vincent Demeester <vincent@sbr.pm> (@vdemeester)
 
+v1.0.0-rc1: git://github.com/containous/traefik-library-image@7eaf03d205ba9bbd6dbcad2675516c33043f79ff
 v1.0.0-beta.704: git://github.com/containous/traefik-library-image@0b0c26da8cb7abc1190d95acbfc5c5e1f986f685
 v1.0.0-beta.404: git://github.com/containous/traefik-library-image@04d09c3cf006f360611fcb178d8537c318100d85
 v1.0.0-beta.392: git://github.com/containous/traefik-library-image@7f7b1e2c7d8ca00ae07e37e64a0e305f7422f6aa
```

Comment:
Any particular reason to keep these older versions listed here?  Removing them here will remove them from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding it on the official build server), but the tag will still be available to users who want it.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)